snipeit.sh

```diff
@@ -86,44 +87,46 @@ vhenvfile () {
 }
 
 perms () {
-		if [ $distro == "debian" ]; then
-		#Change permissions on directories
-		 chmod -R 755 $webdir/$name/storage
-		chmod -R 755 $webdir/$name/storage/private_uploads
-		chmod -R 755 $webdir/$name/public/uploads
-		chown -R www-data:www-data /var/www/$name
-		# echo "* Finished permission changes."
-		else
-		sudo chmod -R 755 $webdir/$name/storage
-		sudo chmod -R 755 $webdir/$name/storage/private_uploads
-		sudo chmod -R 755 $webdir/$name/public/uploads
-		sudo chown -R www-data:www-data /var/www/$name
-		fi
+	chmod_dirs=( "$webdir/$name/storage" )
+	chmod_dirs+=( "$webdir/$name/storage/private_uploads" )
+	chmod_dirs+=( "$webdir/$name/public/uploads" )
+	#Change permissions on directories
+	for chmod_dir in "${chmod_dirs[@]}"
+	do
+		chmod -R 755 "$chmod_dir"
+	done
+}
+    
+log () {
+	eval "$@" |& tee -a /var/log/snipeit-install.log >/dev/null 2>&1
 }
 
 #CentOS Friendly f(x)s
 function isinstalled {
-  if yum list installed "$@" >/dev/null 2>&1; then
-    true
-  else
-    false
-  fi
+	if yum list installed "$@" >/dev/null 2>&1; then
+		true
+	else
+		false
+	fi
 }
 
 if [ -f /etc/lsb-release ]; then
-    distro="$(lsb_release -s -i )"
-    version="$(lsb_release -s -r)"
+	. /etc/lsb-release
```

Comment:
I've tested using stock docker images for Debian and Ubuntu, the command did not exist in both. We should probably just install ```lsb-release``` package.